packages/hardhat-core/test/internal/hardhat-network/provider/node.ts

```diff
@@ -47,7 +47,7 @@ interface ForkPoint {
    */
```

Comment:
I think this comment is outdated now. Maybe rename `blockNumber` to something more explicit like `blockToRun`?

Reply:
`ForkPoint` may also be outdated.